amgut/templates/open-humans.html

```diff
@@ -0,0 +1,79 @@
+{% extends sitebase.html %}
+
+{% from amgut import media_locale %}
+
+{% block head %}
+<style type="text/css">
+  #open-humans {
+    text-align: left;
+    padding: 1em;
+  }
+</style>
+
+<script>
+  var accessToken = '{{ access_token }}';
+
+  var OPEN_HUMANS_URL = '{% raw open_humans_url %}';
+</script>
+
+<script src="{% raw media_locale['SITEBASE'] %}/static/js/open_humans.js"></script>
+{% end %}
+
+{% block content %}
+  <div id="open-humans">
+    <h2>Open Humans Data Export</h2>
+
+    <p>Open Humans is an exciting project that puts participants in charge of
+      data sharing.</p>
+
+    <p>Members of Open Humans can aggregate data from various research studies
```

Comment:
I recommend being more specific, since arguably a lot of things are potentially identifying, especially when combined with other information.

e.g.
This will export to Open Humans: Your raw sequencing files and survey answers.
This will not share: Your name, email address, or other confidential account information.